### PR TITLE
updating mock to return single space string on empty user grade

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
I'm throwing an error, but it's probably not how we throw errors 🤔

### CODE REVIEW

File: src/services/employees.js [hunk 1]
```diff
@@ -374,6 +375,15 @@ const get_persons = async query => {
     const data = await get_employees_by_query(query, get_persons_filters)
     return data.map(emp => {
     emp["employee_profile_url"] = `www.talentmap/profile/public/${emp.first_name}_${emp.last_name}.com`;
+
+    if(!_.isNil(emp.grade_code)) {
+        const re = /\s+/gi;
+        const found = emp.grade_code.match(re);
+        if(found) {
+        console.error('Employee grade an empty string.')
```
Comment (mjoyce91) [inline src/services/employees.js:383]: Use something like this:

```json
res.status(200).send({ Data: null, usl_id: 4000004, return_code: -1 })
```

Comment (elizabeth-jimenez) [reply src/services/employees.js:383]: Thanks Mike! Will do!

File: src/services/employees.js [hunk 2]
```diff
@@ -374,6 +375,15 @@ const get_persons = async query => {
     const data = await get_employees_by_query(query, get_persons_filters)
     return data.map(emp => {
     emp["employee_profile_url"] = `www.talentmap/profile/public/${emp.first_name}_${emp.last_name}.com`;
+
+    if(!_.isNil(emp.grade_code)) {
```
[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply src/services/employees.js:379]: 😢  dang! Ok, will do!

**Comment by elizabeth-jimenez** [comment]
👶 🦶

File: src/services/common.js [hunk 3]
```diff
@@ -132,6 +132,12 @@ const createPositionQuery = (model, tableName, paramPrefix, query, isCount) => {
     Object.keys(query).map(q => {
       const filter = getFilter(q)
       const value = query[q]
+      if (filter.field === 'positions.pos_grade_code') {
```
Comment (mjoyce91) [inline src/services/common.js:135]: Change to

```js
if (get(filter, 'field') === ...
```

so that there isn't an uncaught error when filter is undefined

Comment (elizabeth-jimenez) [reply src/services/common.js:135]: Will do! Thanks Mike!

**Pull request merged**